src/app/shared/utils/lesson-presences.ts

```diff
@@ -21,6 +21,45 @@ export function getIdsGroupedByPerson(
   });
 }
```

Comment:
Wird das `getIdsGroupedByPerson` noch sonst wo verwendet? Sonst könnten wir es auch löschen...

Reply:
Es wird noch in `MyAbsencesReportSelectionService` und `EditAbsencesSelectionService` verwendet.